lms/static/sass/shared-v2/_components.scss

```diff
@@ -0,0 +1,49 @@
+// TODO: tabs should be added to the Pattern Library
+.tabs {
+    @include clearfix();
+    @extend %reset-lists;
```

Comment:
https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order

I was backwards: includes first, then extends. The rest of the Sass property ordering goes out to in using the box model, so, position, margin, border, padding, background, etc.